osu.Game/Graphics/Containers/ExpandingContainer.cs

```diff
@@ -40,21 +40,22 @@ protected ExpandingContainer(float contractedWidth, float expandedWidth)
             RelativeSizeAxes = Axes.Y;
             Width = contractedWidth;
 
-            InternalChild = new OsuScrollContainer
+            InternalChild = CreateScrollContainer().With(s =>
             {
-                RelativeSizeAxes = Axes.Both,
-                ScrollbarVisible = false,
-                Child = FillFlow = new FillFlowContainer
-                {
-                    Origin = Anchor.CentreLeft,
-                    Anchor = Anchor.CentreLeft,
-                    RelativeSizeAxes = Axes.X,
-                    AutoSizeAxes = Axes.Y,
-                    Direction = FillDirection.Vertical,
-                },
-            };
+                s.RelativeSizeAxes = Axes.Both;
+                s.ScrollbarVisible = false;
+            }).WithChild(FillFlow = new FillFlowContainer
+            {
+                Origin = Anchor.CentreLeft,
+                Anchor = Anchor.CentreLeft,
+                RelativeSizeAxes = Axes.X,
+                AutoSizeAxes = Axes.Y,
+                Direction = FillDirection.Vertical,
+            });
         }
 
+        protected virtual OsuScrollContainer CreateScrollContainer() => new OsuScrollContainer();
```

Comment:
Just so that @peppy doesn't yell at me for letting another virtual call in constructor through unscrutinised, does this *need* to be overridable? Can the behaviour this PR is attempting to introduce exist in the base class?

Normally I'd just advise to slap a `[Pure]` attribute on this but the subclass is doing weird games with storing the scroll container to its own field so putting that attribute on would just be a straight up lie, and this is walking a very thin line between "dodgy but fine" and "dodgy and definitely not fine".